src/servers/exporter.ts

```diff
@@ -143,9 +144,14 @@ const main = async () => {
     res.end(await register.metrics())
   })
 
-  server.get("/healthz", async (req, res) => {
-    res.send("OK")
-  })
+  server.get(
+    "/healthz",
+    healthzHandler({
+      checkDbConnectionStatus: true,
+      checkRedisStatus: true, // TODO: set to false when invoices/payment updates are removed from get balance
```

Comment:
What's this comment referring to?

Reply:
exporter only needs db and lnds to get the balances but since we use getBalanceForWallet the update of `pending invoices and payments` are triggered and that functions need redis

Reply:
Last week we updated to `getBalanceForWalletId` so now we no longer need redis check. Updated here https://github.com/GaloyMoney/galoy/pull/795/commits/cf8b2324ae2a1b56207a9d84a9b21eae8b71055a